File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio16.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio16 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Quantos cigarros você fuma por dia?");
        int cigarrosPorDia = entrada.nextInt();
        System.out.println("Há quantos anos você fuma?");
        int anosComoFumante = entrada.nextInt();

        int minutosPerdidosPorDia = cigarrosPorDia * 10;
        int minutosPerdidosPorAno = minutosPerdidosPorDia * 365;
        int minutosPerdidos = minutosPerdidosPorAno * anosComoFumante;
        int diasPerdidos = (minutosPerdidos/60)/24;
        System.out.println("Dias de vida perdidos: " +diasPerdidos);
    }
}

File: guanabara-c/lista01/exercicio16.c
#include <stdio.h>

int main(){
    int cigarros_dia, anos_fumante, total_cigarros, minutos_perdidos, dias_perdidos;

    printf("Quantos cigarros você fuma por dia? ");
    scanf("%d", &cigarros_dia);
    printf("Há quantos anos você fuma? ");
    scanf("%d", &anos_fumante);

    total_cigarros = (cigarros_dia * 365) * anos_fumante;
    minutos_perdidos = 10 * total_cigarros;
    dias_perdidos = (minutos_perdidos/60)/24;
    printf("Dias de vida perdidos: %d", dias_perdidos);

    return 1;
}
